packages/docusaurus-plugin-content-blog/src/index.ts

```diff
@@ -414,5 +420,55 @@ export default function pluginContentBlog(
         }),
       );
     },
+
+    injectHtmlTags() {
+      if (!options.feedOptions) {
+        return;
```

Comment:
I think this will error. Is TS not caching it ?

```suggestion
        return {};
```

Reply:
Oh, blog plugin return type is loosely typed, unlike docs plugin that's why its not throwing TS error. I'll strengthen the typing after this pr is merged

Reply:
I'm gonna send a PR to fix the typing, so you can have better typing.